modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,115 @@
+= Live migration to the OVN-Kubernetes network plugin overview
+
+A _live_ migration process is available when migrating to the OVN-Kubernetes network plugin from the OpenShift SDN plugin. This migration method offers no service interruption, minimizes downtown, and automatically reboots nodes. This method is valuable for deployments types that require constant service availability, such as Azure Red Hat OpenShift (ARO), Red Hat OpenShift Dedicated, or Red Hat OpenShift Service on AWS.
+
+== Considerations for migrating to the OVN-Kubernetes network plugin
+
+* With the live migration, there is a Maximum Transmission Unit (MTU) mismatch between OpenShift SDN and OVN-Kubernetes, and MTU values might differ. The Cluster Network Operator (CNO) handles updating the routable MTU to ensure both CNIs share the same overlay MTU. 
+
+* When the live migration procedure begins, multitenant isolation is permanently disabled. Multitenant isolation is only supported on OpenShift SDN. 
+
+* During the live migration, when the cluster is running both OVN-Kubernetes and OpenShift SDN, multicast and egress IP are temporarily disabled for both CNIs. The egress firewall remains functional during this process.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* Live migration does not work in clusters without the `dedicated-admin` namespace. This namespace is only created by _managed_ clusters. 
```

Comment:
id remove this implementation detail about namespaces being present. 
Just that live migration only support on  managed clusters excluding hypershift based clusters.